### PR TITLE
Assimilate 1.2.0 -> rally

### PR DESCRIPTION
**Pull request by jeffmay**
@vodopis @chrissalij-r

### CODE REVIEW

**Comment by scalp42** [comment]
LGTM :100:

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+<a href='https://travis-ci.org/jeffmay/play-json-ops'>
```
Comment (chrissalij-r) [inline README.md]: This isn't markdown :/

**Comment by chrissalij-r** [comment]
LGTM

**Comment by vodopis** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (jeffmay) [inline README.md]: It's supported by github markdown :)

**Comment by jeffmay** [comment]
Released as v1.2.0: https://rally-jenkins.werally.in/job/Release_play-json-ops/2/

**Pull request merged**